From: undef <gitlab@undef.tools>
Date: Mon, 11 Jul 2022 05:47:50 +0000
Subject: arm64: configs: attempt to turn on ROCKCHIP_ISP1=y

This currently fails due to module dependancies. Required for display to
start on boot
---
 arch/arm64/configs/rockchip_defconfig | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 1626a57..0df2c48 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -1347,7 +1347,7 @@ CONFIG_RC_LOOPBACK=m
 CONFIG_MEDIA_CEC_SUPPORT=y
 CONFIG_USB_PULSE8_CEC=m
 CONFIG_USB_RAINSHADOW_CEC=m
-CONFIG_MEDIA_SUPPORT=m
+CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_SUPPORT_FILTER=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 CONFIG_MEDIA_CAMERA_SUPPORT=y
@@ -1356,6 +1356,8 @@ CONFIG_MEDIA_DIGITAL_TV_SUPPORT=y
 CONFIG_MEDIA_RADIO_SUPPORT=y
 CONFIG_MEDIA_SDR_SUPPORT=y
 CONFIG_MEDIA_PLATFORM_SUPPORT=y
+CONFIG_VIDEO_DEV=y
+CONFIG_MEDIA_CONTROLLER=y
 CONFIG_VIDEO_ADV_DEBUG=y
 CONFIG_V4L2_FLASH_LED_CLASS=m
 CONFIG_MEDIA_USB_SUPPORT=y
@@ -1435,10 +1437,10 @@ CONFIG_USB_KEENE=m
 CONFIG_USB_RAREMONO=m
 CONFIG_USB_MA901=m
 CONFIG_V4L_PLATFORM_DRIVERS=y
-CONFIG_VIDEO_ROCKCHIP_ISP1=m
 CONFIG_V4L_MEM2MEM_DRIVERS=y
 CONFIG_VIDEO_MEM2MEM_DEINTERLACE=m
-CONFIG_VIDEO_ROCKCHIP_RGA=m
+CONFIG_VIDEO_ROCKCHIP_RGA=y
+CONFIG_VIDEO_ROCKCHIP_ISP1=y
 CONFIG_SMS_SDIO_DRV=m
 CONFIG_VIDEO_TVAUDIO=m
 CONFIG_VIDEO_TDA7432=m
@@ -1459,10 +1461,13 @@ CONFIG_VIDEO_ADV7170=m
 CONFIG_VIDEO_ADV7175=m
 CONFIG_VIDEO_UPD64031A=m
 CONFIG_VIDEO_UPD64083=m
-CONFIG_VIDEO_IMX258=m
+CONFIG_VIDEO_IMX258=y
 CONFIG_VIDEO_OV7670=m
+CONFIG_VIDEO_OV8858=y
 CONFIG_VIDEO_SR030PC30=m
+CONFIG_VIDEO_DW9714=y
 CONFIG_VIDEO_NOON010PC30=m
+CONFIG_MEDIA_TUNER=y
 CONFIG_DRM=y
 CONFIG_DRM_DP_AUX_CHARDEV=y
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
